inputs/prod_inner.json

```diff
@@ -66,6 +66,13 @@
     },
     {
       "args": {
+        "koji_root": "{{KOJI_ROOT}}",
+        "koji_hub": "{{KOJI_HUB}}"
+      },
+      "name": "fetch_maven_artifacts"
```

Comment:
Do we want `"required": false`, to retain compatibility with atomic-reactor-1.6.18?